axlearn/common/utils_spmd.py

```diff
@@ -53,7 +53,7 @@ def setup(
         if initialization_timeout is not None:
             init_kwargs["initialization_timeout"] = initialization_timeout
 
-        if jax_backend == "tpu":
+        if jax_backend in ("tpu", "proxy"):
```

Comment:
Add a comment that "proxy" is for the pathways codepath?

Reply:
ty! Added a comment